doc/man7/Ed25519.pod

```diff
@@ -25,7 +25,7 @@ The message to sign or verify must be passed using the one-shot
 EVP_DigestSign() and EVP_DigestVerify() functions.
 
 When calling EVP_DigestSignInit() or EVP_DigestVerifyInit(), the
-digest parameter B<MUST> be set to B<NULL>.
+digest B<type> parameter B<MUST> be set to B<NULL>.
```

Comment:
I am not a native English speaker, but it seems if you want to address the parameter's name, place the name after the word 'parameter' sounds common - something like 'the digest parameter type MUST be set to NULL'.

Let's wait for others' opinions...

Reply:
Not a native speaker either, but having seen both forms...  Anyway, we have an actual English speaker that has approved this, I suspect we can see that as an "ok" sign :smiley: